tracer/dependabot/Datadog.Dependabot.Integrations.csproj

```diff
@@ -145,18 +145,18 @@
 
     <!-- Integration: Microsoft.AspNetCore.Server.Kestrel.Core -->
     <!--    Assembly: Microsoft.AspNetCore.Server.Kestrel.Core -->
-    <!-- Latest package https://www.nuget.org/packages/Microsoft.AspNetCore.Server.Kestrel.Core/2.2.0 -->
-    <PackageReference Include="Microsoft.AspNetCore.Server.Kestrel.Core" Version="2.2.0" />
+    <!-- Latest package https://www.nuget.org/packages/Microsoft.AspNetCore.Server.Kestrel.Core/2.3.0 -->
+    <PackageReference Include="Microsoft.AspNetCore.Server.Kestrel.Core" Version="2.3.0" />
 
     <!-- Integration: Microsoft.AspNetCore.Session -->
     <!--    Assembly: Microsoft.AspNetCore.Session -->
-    <!-- Latest package https://www.nuget.org/packages/Microsoft.AspNetCore.Session/2.2.0 -->
-    <PackageReference Include="Microsoft.AspNetCore.Session" Version="2.2.0" />
+    <!-- Latest package https://www.nuget.org/packages/Microsoft.AspNetCore.Session/2.3.0 -->
+    <PackageReference Include="Microsoft.AspNetCore.Session" Version="2.3.0" />
 
     <!-- Integration: Microsoft.AspNetCore.StaticFiles -->
     <!--    Assembly: Microsoft.AspNetCore.StaticFiles -->
-    <!-- Latest package https://www.nuget.org/packages/Microsoft.AspNetCore.StaticFiles/2.2.0 -->
-    <PackageReference Include="Microsoft.AspNetCore.StaticFiles" Version="2.2.0" />
+    <!-- Latest package https://www.nuget.org/packages/Microsoft.AspNetCore.StaticFiles/2.3.0 -->
+    <PackageReference Include="Microsoft.AspNetCore.StaticFiles" Version="2.3.0" />
```

Comment:
Urgh, oh yeah, the .NET Core 2.1 rename to 2.3... fun... I wonder how we should go about testing this...